pgjdbc/src/test/java/org/postgresql/test/jdbc2/TimestampTest.java

```diff
@@ -577,30 +578,54 @@ private void timestampTestWOTZ() throws SQLException {
       assertNotNull(t);
       assertTrue(t.equals(TS1WOTZ));
 
+      tString = rs.getString(1);
+      assertNotNull(tString);
+      assertTrue(tString.equals(TS1WOTZ_PGFORMAT));
```

Comment:
Thanks, you're right it is indeed better. I will make the change in the fix commit. 

I was also tempted to refactor the surrounding code in the test to instead iterate through the test cases with a test function for brevity. I was planning on doing that in a subsequent refactoring commit after tests look good--for sake of keeping the fix changes minimal.